freqtrade/fiat_convert.py

```diff
@@ -0,0 +1,161 @@
+import logging
+import time
+from pymarketcap import Pymarketcap
+
+logger = logging.getLogger(__name__)
+
+
+class PairConvertion():
```

Comment:
I don't think `Convertion` is a word. The name is pretty vague, I'd vote for something more explicit, like `CryptoToFiat` or `CryptoToFiatConverter`.

Reply:
Thank you for this comment. You are right for the name convention. I had a lack of imagination. This is updated